### PR TITLE
Fixing Examples and typename picking

### PR DESCRIPTION
**Pull request by bannanc**
I'm addressing issues #211 and #210

### CODE REVIEW

**Comment by bannanc** [comment]
@davidlmobley Could you check that you approve of these changes and merge if you do? (assuming tests pass)

File: examples/smarty_smiluations/README.md
Comment (davidlmobley) [inline examples/smarty_smiluations/README.md]: "implement"

File: examples/smarty_smiluations/README.md
Comment (davidlmobley) [inline examples/smarty_smiluations/README.md]: "example outputs"

**Review by davidlmobley — APPROVED** [review]
@bannanc - You have an examples directory named `smarty_smiluations`. Also, the README.md for the examples references `smart/data`. 

Previously, John had requested some example output be noted and commented on in the `README.md`, which is I think a good idea. This isn't always super intuitive to look at, and he'd gone on the GItHub repo to try and show someone what was going on and had trouble finding a good example. I think your changes remove this from the README.md. Could you add some back in? I'm not talking about a full log file, but giving an example (perhaps) of where something starts on AlkEthOH, then how it gets to 100% at some point, and briefly commenting on the differences between them. You could comment on a move as well. 

After that I think it will be good to merge (and you can subsequently merge when tests pass).

**Comment by bannanc** [comment]
@davidlmobley I added more detailed output and commentary for one simulation, I worried it would get way too long if I did that for all of them. 
Is that sufficient, or should I do the same for the other two examples?

**Comment by davidlmobley** [comment]
One example is fine. Thanks!

**Pull request merged**
